### DISCUSSION TITLE
How to enable compare mode

### DISCUSSION THREAD

**Discussion by dongwonseong** [tradingview/lightweight-charts#985] [Q&A] [answered]
Hello.

I want to make a comparison chart of two or more stocks with a lightweight chart, not a charting library.

Is it possible to implement a stock comparison chart with lightweight chart?

If possible, where can i find it?

thank you

**Comment by timocov** [comment, marked as answer]
You need to change the mode of a price scale to percentage, see https://tradingview.github.io/lightweight-charts/docs/api/enums/PriceScaleMode#percentage
